src/select-menu/src/SelectMenuContent.js

```diff
@@ -36,26 +42,48 @@ export default class SelectMenuContent extends PureComponent {
   static defaultProps = {
     options: [],
     hasTitle: true,
-    hasFilter: true
+    hasFilter: true,
+    titleView: ({ close, title, headerHeight }) => (
```

Comment:
Nit: I would pull this out into a separate component so the defaultProps are easier to scan.

But this is exactly what I was thinking of!

Reply:
Whoo! ok, I'll do that.